internal/transform/operation.go

```diff
@@ -94,6 +97,61 @@ func addAssetDetailsToOperationDetails(operationDetails *Details, asset xdr.Asse
 	return nil
 }
 
+func addTrustLineFlagToDetails(operationDetails *Details, f xdr.TrustLineFlags, prefix string) {
+	var (
+		n []int32
+		s []string
+	)
+
+	if f.IsAuthorized() {
+		n = append(n, int32(xdr.TrustLineFlagsAuthorizedFlag))
+		s = append(s, "authorized")
+	}
+
+	if f.IsAuthorizedToMaintainLiabilitiesFlag() {
+		n = append(n, int32(xdr.TrustLineFlagsAuthorizedToMaintainLiabilitiesFlag))
+		s = append(s, "authorized_to_maintain_liabilities")
+	}
+
+	if f.IsClawbackEnabledFlag() {
+		n = append(n, int32(xdr.TrustLineFlagsTrustlineClawbackEnabledFlag))
+		s = append(s, "clawback_enabled")
+	}
+
+	switch prefix {
+	case "set":
+		operationDetails.SetFlags = n
+		operationDetails.SetFlagsString = s
+
+	case "clear":
+		operationDetails.ClearFlags = n
+		operationDetails.ClearFlagsString = s
+	}
+
+}
+
+func addLedgerKeyToDetails(operationDetails *Details, ledgerKey xdr.LedgerKey) error {
```

Comment:
(1 more ❓) for my own knowledge what does ledgerKey represent?

Reply:
My understanding is that it is a struct comprising the `LedgerEntryType` with the unique identifier per `Type`.  Each state table in `stellar-core` has a unique ID per state table and the `LedgerKey` lets you access those IDs.

For example, if the `LedgerEntryType` is Account, you would be able to access the `AccountID`. Similarly, Claimable Balance would return the `BalanceID` etc